2014-04-16  Richard Henderson  <rth@redhat.com>
 
	* ports/sysdeps/unix/alpha/sysdep.h (__pointer_chk_guard): Remove
	const from the non-libc, non-ldso copy.

--- a/ports/sysdeps/unix/alpha/sysdep.h
+++ b/ports/sysdeps/unix/alpha/sysdep.h
@@ -377,7 +377,7 @@ extern uintptr_t __pointer_chk_guard_local attribute_relro attribute_hidden;
 #  define PTR_MANGLE(var) \
 	(var) = (__typeof (var)) ((uintptr_t) (var) ^ __pointer_chk_guard_local)
 # else
-extern const uintptr_t __pointer_chk_guard attribute_relro;
+extern uintptr_t __pointer_chk_guard attribute_relro;
 #  define PTR_MANGLE(var) \
 	(var) = (__typeof(var)) ((uintptr_t) (var) ^ __pointer_chk_guard)
 # endif
